firedrake/plot.py

```diff
@@ -0,0 +1,121 @@
+from __future__ import absolute_import
+import numpy as np
+from ufl import cell
+from firedrake import Function, SpatialCoordinate
+
+
```

Comment:
To control what is exposed in the public API (when doing `from firedrake.plot import *`) can you define `__all__`?  E.g., if you only want to expose the `plot` function:

```python
__all__ = ("plot", )
```

Other functions are still available if you `import firedrake.plot` as `firedrake.plot.XXX`.